src/utils/headersToSchema.ts

```diff
@@ -14,14 +20,23 @@ export function headersToSchema(
     `Defines headers for ${operationFullName} operation.`
   );
 
-  headers.forEach(({ header, schema }) => {
+  headers.forEach(({ header, language, schema }) => {
     if (!headersSchema.properties) {
       headersSchema.properties = [];
     }
 
-    const { description } = getLanguageMetadata(schema.language);
+    if (!language) {
```

Comment:
Now `language` is mandatory so you won't need to check for it anymore :)

Reply:
Nice! Thanks!